packages/waku/src/router/client.ts

```diff
@@ -223,7 +223,7 @@ function InnerRouter() {
       if (typeof path === 'string') {
         url.pathname = path;
       }
-      if (searchParams) {
+      if (searchParams?.size) {
         url.search = '?' + searchParams.toString();
```

Comment:
Nice catch. That looks like a good change.

But, if you are familiar with `url.search`, do you know if it's common or recommended to add a search string without prepending `?`?

Here's an idea:
```suggestion
      if (searchParams) {
        url.search = searchParams.toString();
```

Reply:
Yes, that's a better one, updated

Reply:
thanks.